xAPI.md

```diff
@@ -2778,7 +2778,14 @@ track learning records.
 
 __Note:__ In all of the example endpoints given in the specification, 
 "http://example.com/xAPI/" is the example base endpoint of the LRS. All other IRI 
-syntax after this represents the particular endpoint used.
+syntax after this represents the particular endpoint used. A full list of endpoints
+is included in [Appendix F: Table of All Endpoints](#AppendixF).
+
+###### Requirements
+
+* The LRS MUST support all of the endpoints described in [this section](#datatransfer). (__Note:__ This requirement *does not* apply to 
```

Comment:
I recommend we modify to "This requirement only applies to the OAuth endpoints described in ... if the LRS implements OAuth 1.0"